packages/11ty/_plugins/transforms/pdf/README.md

```diff
@@ -0,0 +1,60 @@
+## Tranfrom for PDF
+
+The PDF module implements an [Eleventy `transform` function](https://www.11ty.dev/docs/config/#transforms) that prepares the static output for input to the [`Paged.js`](https://pagedjs.org) library. The HTML output from static rendering is output to a combined HTML file that is the input to the [`Paged.js`](https://pagedjs.org) commands.
+
+## Reading the HTML output files
+
+The transform function is called for each template, with the output HTML and output path as arguments, at the end of the build step before the individual page content is written to file on disk.
+
+### Selecting the content
+
+The transfrom uses the [`JSDOM` library](https://github.com/jsdom/jsdom) to create an HTML `document` from the rendered output and `querySelect` a `section` element that contains the content. The `section` containing the content is selected by the `data-ouput-path` attribute applied to the wrapping `section` element rendered by the `_layouts/base.11ty.js` template.
+
+Below is simplified example of the `content` argument passed to the transform function:
+
+```HTML
+<!DOCTYPE html>
+<html>
+  <head>
+    <-- omitted for example -->
+  </head>
+  <body>
+    <header>
+      <-- omitted for example -->
+    </header>
+    <main>
+      <nav>
+        <-- omitted for example -->
+      </nav>
+      <section data-attribute="example/page.html">
+        <p>This is the combined output of the `content` directory template file and the applied layouts.</p>
+      </section>
+    </main>
+    <footer>
+      <-- omitted for example -->
+    </footer>
+  </body>
+</html>
+```
```

Comment:
```suggestion
```HTML
<!DOCTYPE html>
<html>
  <head>
    <-- omitted -->
  </head>
  <body>
    <header>
      <-- omitted -->
    </header>
    <main>
      <nav>
        <-- omitted -->
      </nav>
      <section data-attribute="example/page.html">
        <p>This is the combined output of the `content` directory template file and the applied layouts.</p>
      </section>
    </main>
    <footer>
      <-- omitted for example -->
    </footer>
  </body>
</html>
```
```